tidb-cloud/changefeed-sink-to-cloud-storage.md

```diff
@@ -0,0 +1,82 @@
+
+4. Customize **Table Filter** to filter the tables that you want to replicate. For the rule syntax, refer to [table filter rules](https://docs.pingcap.com/tidb/stable/ticdc-filter#changefeed-log-filters).
+![the table filter of changefeed](/media/tidb-cloud/changefeed/sink-to-s3-02-table-filter.jpg)
+
+    - **Filter rules**: You can configure filter rules in this column. The default rule is *. *, which means all tables will be replicated. When you add a new rule, TiDB Cloud will query all the tables in TiDB and display only the tables that match the specified rules in the right-hand box.
+    - **Tables with valid keys**: This column displays the tables that have valid keys, including primary keys or unique indexes.
+    - **Tables without valid keys**: This column shows tables that lack primary keys or unique keys. These tables present a challenge during replication because the absence of a unique identifier can result in inconsistent data when handling duplicate events downstream. To ensure data consistency, it is recommended to add unique keys or primary keys to these tables before initiating replication. Alternatively, you can employ filter rules to exclude these tables. For example, you can exclude the table `test.tbl1` by using the rule "!test.tbl1".
+
+
+5. In the **Start Replication Position** area, select one of the following replication positions:
+    - Start replication from now on
+    - Start replication from a specific [TSO](https://docs.pingcap.com/tidb/stable/glossary#tso)
+    - Start replication from a specific time
+
+6. In the **Data Format** area, select either the `CSV` or `Canal-JSON` format.
+
+<SimpleTab>
+<div label="Configure CSV Format">
+To configure the `CSV` format:
+
+![the data format of CSV](/media/tidb-cloud/changefeed/sink-to-s3-02-data-format-csv-conf.jpg)
+
+- **Data Separator**: To rotate data based on the year, month, and day, or choose not to rotate at all.
+- **Delimiter**: Specify the character used to separate values in the CSV file. The comma (,) is the most commonly used delimiter.
+- **Quote**: Specify the character used to enclose values that contain the delimiter character or special characters. Typically, double quotes (") are used as the quote character.
+- **Null/Empty Values**: Specify how null or empty values are represented in the CSV file. This can be important for proper handling and interpretation of the data.
+- **Include Commit Ts**
+
+</div>
+<div label="Configure Canal-JSON Format">
+Canal-JSON is a plain JSON text format, to configure it: 
+
+![the data format of Canal-JSON](/media/tidb-cloud/changefeed/sink-to-s3-02-data-format-canal-json.jpg)
+
+- **Data Separator**: To rotate data based on the year, month, and day, or choose not to rotate at all.
+- **Enable TiDB Extention** 
+
+</div>
+</SimpleTab>
```

Comment:
```suggestion
    <SimpleTab>
    <div label="Configure CSV Format">
    To configure the `CSV` format:
    
    ![the data format of CSV](/media/tidb-cloud/changefeed/sink-to-s3-02-data-format-csv-conf.jpg)
    
    - **Data Separator**: To rotate data based on the year, month, and day, or choose not to rotate at all.
    - **Delimiter**: Specify the character used to separate values in the CSV file. The comma (,) is the most commonly used delimiter.
    - **Quote**: Specify the character used to enclose values that contain the delimiter character or special characters. Typically, double quotes (") are used as the quote character.
    - **Null/Empty Values**: Specify how null or empty values are represented in the CSV file. This can be important for proper handling and interpretation of the data.
    - **Include Commit Ts**
    
    </div>
    <div label="Configure Canal-JSON Format">
    Canal-JSON is a plain JSON text format, to configure it: 
    
    ![the data format of Canal-JSON](/media/tidb-cloud/changefeed/sink-to-s3-02-data-format-canal-json.jpg)
    
    - **Data Separator**: To rotate data based on the year, month, and day, or choose not to rotate at all.
    - **Enable TiDB Extention** 
    
    </div>
    </SimpleTab>
```